cg/meta/demultiplex/validation.py

```diff
@@ -35,11 +36,16 @@ def is_lane_in_fastq_file_name(sample_fastq: Path) -> bool:
     return bool(re.search(r"_L\d+", sample_fastq.name))
 
 
-def is_sample_id_in_directory_name(directory: Path) -> bool:
+def is_sample_id_in_directory_name(directory: Path, sample_id: str) -> bool:
     """Validate that directory name contains the sample id formatted as Sample_<sample_id>"""
     return bool(re.search(r"Sample_\w+", directory.name))
```

Comment:
Where do we actually check that sample_id is in the directory name? Seems as if the provided regexp does check for it.

Reply:
Good catch!